src/common/store/session/actions.ts

```diff
@@ -50,6 +50,7 @@ export const actions: ActionTree<SessionState, RootState> = {
     };
     const rLogin = state.rLoginInstance === undefined ? new RLogin({
       cacheProvider: false,
+      defaultTheme: 'dark',
```

Comment:
The defaultTheme implies in every time use dark mode?

Reply:
Yes, unless user change it in rlogin popup. We need to find a way to sync this theme and app's theme.